### PR TITLE
[cisco_ftd] Add rule, device and host ECS fields

### PR DESCRIPTION
**Pull request by LaZyDK**
## What does this PR do?

Adds test data for Cisco ISE pxGrid Endpoint Profile data.
Adds `rule`, `host` and `device` ECS fields.
Removes empty fields.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

File: packages/cisco_ftd/data_stream/log/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -2281,27 +2366,20 @@ processors:
   - script:
       lang: painless
       description: This script processor iterates over the whole document to remove fields with null values.
```
Comment (efd6) [inline packages/cisco_ftd/data_stream/log/elasticsearch/ingest_pipeline/default.yml:2368]:
```suggestion
      description: This script processor iterates over the whole document to remove fields with null or empty values.
```

File: packages/cisco_ftd/data_stream/log/elasticsearch/ingest_pipeline/default.yml [hunk 2]
```diff
@@ -1986,6 +1989,67 @@ processors:
...
+  - gsub: 
+        field: _temp_.host.type
+        pattern: "-"
+        replacement: " "
+        if: "ctx._temp_?.host?.type != null"
+  - gsub: 
+        field: _temp_.host.type
+        pattern: "_"
+        replacement: " "
+        if: "ctx._temp_?.host?.type != null"
```
Comment (efd6) [inline packages/cisco_ftd/data_stream/log/elasticsearch/ingest_pipeline/default.yml:2031]:
```suggestion
  - gsub: 
        field: _temp_.host.type
        pattern: "(?:Workstation|[-_])"
        replacement: " "
        if: "ctx._temp_?.host?.type != null"
```

File: packages/cisco_ftd/data_stream/log/fields/fields.yml [hunk 3]
```diff
@@ -3,85 +3,105 @@
   fields:
     - name: message_id
       type: keyword
-      description: |
+      description: >
```
Comment (efd6) [inline packages/cisco_ftd/data_stream/log/fields/fields.yml:6]: Is there a reason for making these changes?

(note)
```
$ rg -c 'description: |' | cut -d: -f2 | paste -sd+ - | bc
3413140
$ rg -c 'description: >' | cut -d: -f2 | paste -sd+ - | bc
5275
```

Comment (LaZyDK) [reply packages/cisco_ftd/data_stream/log/fields/fields.yml:6]: For consistency only.

File: packages/cisco_ftd/data_stream/log/elasticsearch/ingest_pipeline/default.yml (hunk 2 above)
Comment (LaZyDK) [reply packages/cisco_ftd/data_stream/log/elasticsearch/ingest_pipeline/default.yml:2031]: I will try this and correct the rest of them if it works as intended.

Comment (LaZyDK) [reply packages/cisco_ftd/data_stream/log/elasticsearch/ingest_pipeline/default.yml]: Working as intended - accepted as is :)

File: packages/cisco_ftd/data_stream/log/fields/fields.yml (hunk 3 above)
Comment (efd6) [reply packages/cisco_ftd/data_stream/log/fields/fields.yml:6]: The reverse change would improve consistency from the stats above.

Comment (efd6) [reply packages/cisco_ftd/data_stream/log/fields/fields.yml:6]: Thank you.

**Review by efd6 — APPROVED** [review]
Thanks

**Pull request merged**